src/main/drivers/light_led.h

```diff
@@ -62,6 +40,22 @@ PG_DECLARE(statusLedConfig_t, statusLedConfig);
 #define LED2_OFF                 ledSet(2, false)
 #define LED2_ON                  ledSet(2, true)
 
+// use dummy functions for unittest
+#if defined(UNIT_TEST) || defined(USE_VIRTUAL_LED)
```

Comment:
Should unit tests just define virtual led?

Reply:
Good idea. 

Normally unittest would use led stubs (light_led.c is not compiled in). UNIT_TEST is trick to reuse this header.
One consequence is that unittesting leds is not possible.